### ISSUE TITLE
LOS tool tweaks

### ISSUE THREAD

**Issue by JonhyBurner** [MegaMek/megamek#2463]
Shouldn't the LOS tool take into account whether the POV is from a Mech or a Vehicle? In the below situation, checking the "Is Mech" square makes no changes to the POV, stating that is blocked by woods. But if I add 1 height (since it is a Mech), the POV is now just obscured by 1 light woods, just like my prone, no-leg Javelin can attest.

The problem is that although the tool works properly when clicking on actual Mechs, when clicking on empty hexes for planning moves, the "Is Mech" box does nothing. The tool simply uses the values from the last use.
I could swear this was behaving differently in the past, and that I didn't have to manually input the Mech height.

![LOS ground level](https://user-images.githubusercontent.com/67933202/100803427-0c4ffb00-3423-11eb-9039-b9d140e24c45.jpg)

![LOS mech level](https://user-images.githubusercontent.com/67933202/100803436-107c1880-3423-11eb-903b-605a7300ed43.jpg)

**Comment by HammerGS** [comment]
Duplicate of #3198
